script/config/podman-compose.yml

```
@@ -47,6 +49,7 @@ services:
     container_name: tls-redis-master
     network_mode: host
     volumes:
+      - ./run:/var/run:z
```

relabeling the whole `/var/run` doesn't seem right. Maybe it doesn't matter much but better create a subdir for the socket. e.g.
```suggestion
      - ./run:/var/run/redis:z
```